app/views/catalog/_search_form.html.erb

```diff
@@ -20,4 +20,4 @@
   <% end %>
 
 
-<%= link_to 'More options', advanced_search_path(params), :class=>'advanced_search'%>
\ No newline at end of file
+<%= link_to 'More options', advanced_search_path(params.symbolize_keys), :class=>'advanced_search'%>
```

Comment:
Totally not related to this PR, dude!

:+1: all the same (I'm assuming this fixes those awful deprecation warnings)